rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
+    hamm_dist = 0
+    for i in range(len(a)):
+        if a[i] != b[i]:
+            hamm_dist += 1
```

Comment:
Challenge: solve this using `sum` a list comprehension. It can be done!

Hint: https://docs.python.org/2/library/functions.html#zip